app/share.tsx

```diff
@@ -141,7 +141,7 @@ class Root extends React.Component<{}, IState> {
 	setTheme = (newTheme = {}) => {
 		// change theme state
 		this.setState(
-			prevState => newThemeState(prevState, newTheme as IThemePreference),
+			prevState => newThemeState(prevState.themePreferences, newTheme as IThemePreference),
```

Comment:
Why?

Reply:
https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4401/files#diff-b788f0c6b6e92a2a43dedd61dd4e596b4b0f981e30bcfd88e2f5d30b2de3a7dbR39

Because I changed the function newThemeState to receive only the property required, not the state completely.